### PR TITLE
removed name Guliza

### PR DESCRIPTION
**Pull request by GulizaS**

### CODE REVIEW

File: src/main/java/ProQAList.java [hunk 1]
```diff
@@ -11,7 +11,7 @@ public static void main(String[] args) {
     public static void printStudentsName(){
 
         List<String> names = new ArrayList<String>();
-        names.add("Guliza");
```
Comment (SlavaErgonov) [inline src/main/java/ProQAList.java]: Where is my name?

Comment (AntonSemenov87) [inline src/main/java/ProQAList.java]: my comment here

File: src/main/java/ProQAList.java [hunk 2]
```diff
@@ -11,7 +11,7 @@ public static void main(String[] args) {
...
+
```
Comment (aybukego) [inline src/main/java/ProQAList.java]: please review your changes

File: src/main/java/ProQAList.java (hunk 1 above)
Comment (GulizaS) [reply src/main/java/ProQAList.java]: I'm working on my name ticket. You can check with the lead about that issue.

Comment (GulizaS) [reply src/main/java/ProQAList.java]: Thanks for you comment Anton

File: src/main/java/ProQAList.java (hunk 2 above)
Comment (GulizaS) [reply src/main/java/ProQAList.java]: fixed. thanks !

**Pull request merged**